test/functional/feature_config_args.py

```diff
@@ -145,11 +146,61 @@ def test_networkactive(self):
             self.start_node(0, extra_args=['-nonetworkactive=1'])
         self.stop_node(0)
 
+    def test_seed_peers(self):
+        self.log.info('Test seed peers')
+        default_data_dir = self.nodes[0].datadir
+
+        # No peers.dat exists and -dnsseed=1
+        # We expect the node will use DNS Seeds, but Regtest mode has 0 DNS seeds
+        # So after 60 seconds, the node should fallback to fixed seeds (this is a slow test)
+        assert not os.path.exists(os.path.join(default_data_dir, "peers.dat"))
+        start = time.time()
+        with self.nodes[0].assert_debug_log(expected_msgs=["Loaded 0 addresses from peers.dat",
+                "0 addresses found from DNS seeds", "Adding fixed seeds as 60 seconds have passed and addrman is empty"],
+                timeout=80):
```

Comment:
```suggestion
        with self.nodes[0].assert_debug_log(expected_msgs=[
                "Loaded 0 addresses from peers.dat",
                "0 addresses found from DNS seeds",
                "Adding fixed seeds as 60 seconds have passed and addrman is empty",
            ],
                timeout=80):
```
(and similarly below)

Reply:
Done.